autoload/lsp/omni.vim

```diff
@@ -194,14 +195,21 @@ function! lsp#omni#get_vim_completion_item(item, ...) abort
                 \ 'dup': 1,
                 \ 'kind': l:kind}
 
+    " check support user_data.
+    if !s:is_user_data_support && g:lsp_text_edit_enabled
+        echohl WarningMsg
+        echom 'textEdit support on omni complete requires Vim 8.0 patch 1493 or later(please check g:lsp_text_edit_enabled)'
```

Comment:
can you also add this to `lsp#log()`

Reply:
Added 728e60b9b437767163c5a426c7a943c7e64a2c05.